frontend/src/Buildings/index.jsx

```diff
@@ -1,23 +1,29 @@
 import React, { useEffect, useState } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import {
-  View, Text, ScrollView,
+  View, ScrollView,
   StyleSheet, ActivityIndicator,
 } from 'react-native';
-import { CardView } from '../common/components';
-import BuildingItem from './buildingItem';
-import AddBuildingItem from './addBuildingItem';
+
 import { fetchBuildings, addBuildingSuccess } from './actionCreator';
+
 import townhallIcon from '../../assets/buildings/townhall.png';
 import academyIcon from '../../assets/buildings/academy.png';
 import farmIcon from '../../assets/buildings/factory.png';
 import mineIcon from '../../assets/buildings/mine.png';
 import addFarmIcon from '../../assets/buildings/addFarm.png';
 import addMineIcon from '../../assets/buildings/addMine.png';
 import addAcademyIcon from '../../assets/buildings/addAcademy.png';
+
 import Colors from '../common/colors';
+import { CardView } from '../common/components';
+
+import BuildingItem from './buildingItem';
+import AddBuildingItem from './addBuildingItem';
 import OneBuilding from '../OneBuilding';
 
+import ErrorHandlerPage from '../ErrorHandlerPage';
```

Comment:
```suggestion
import ErrorPopup from '../ErrorPopup';
```
I think this is a kind of popup, not a page.